sdk/dotnet/AccessAnalyzer/Analyzer.cs

```diff
@@ -48,7 +48,7 @@ public partial class Analyzer : Pulumi.CustomResource
         /// Key-value map of resource tags.
         /// </summary>
         [Output("tags")]
-        public Output<ImmutableDictionary<string, object>?> Tags { get; private set; } = null!;
```

Comment:
These tags changes have been approved to be brought in by @lukehoban - this is due to the revert we did before and has since broken tagging mechanisms